src/Bicep.Decompiler/UniqueNamingResolver.cs

```diff
@@ -68,9 +68,22 @@ private static string EscapeIdentifier(string identifier)
                 {
                     return null;
                 }
-                
-                // TODO technically a naming clash is still possible here but unlikely
-                nameByType[nameType] = $"{desiredName}_{GetNamingSuffix(nameType)}";
+
+                if (nameType == NameType.Output)
```

Comment:
>nameType [](start = 20, length = 8)

Worth adding a decompiler test for this change?

Reply:
The changes to the src/Bicep.Decompiler.IntegrationTests/Working/namingclashes/main.bicep baseline assert this. Or were you thinking a specific code test?